src/app/pages/unuploaded-tests/unuploaded-tests.page.ts

```diff
@@ -57,12 +59,14 @@ export class UnuploadedTestsPage implements OnInit {
             new Date(a.slotData.slotDetail.start).getTime() - new Date(b.slotData.slotDetail.start).getTime())),
       ),
     };
+    this.pageState.unSubmittedTestSlots$.subscribe((data) => {
+      this.slots = data.map((i) => i.slotData);
+    });
```

Comment:
We should add this into the `this.pageState` and using the async pipe inside the template as it auto manages subscriptions.